server/services/content-types.js

```diff
@@ -136,6 +196,14 @@ const getUpdatableRelations = (model) => {
     ) {
       result.push(key);
     }
+
+    if (attributes[key].type === "dynamiczone") {
+      for (const comp of attributes[key].components) {
+        const model = strapi.getModel(comp);
+        const relations = getUpdatableRelations(model);
+        relations && result.push({ [key]: relations });
+      }
+    }
```

Comment:
@ShipleyC95 

I also added this here

`    if (attributes[key].type === "component") {
      const model = strapi.getModel(attributes[key].component);
      const relations = getUpdatableRelations(model);
      relations && result.push({ [key]: relations });
    }`
    
  But then i also need to update the generation of populate statement.

Reply:
Yeah, looking at refactoring the generate statement now to account for nested objects, will hopefully be able to post soon

Reply:
I spent the past hour trying to refactor it but could not figure it out. I will try tomorrow if you are not faster to do it.

Reply:
Please, keep the branch updated if you can :)